### PR TITLE
Refactor all test methods to use @test annotation

### PR DESCRIPTION
**Pull request by daisy1754**
# Description
Refactor all test methods to use @test annotation and remove test prefixes

verified we have same number of running tests after my change

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [x] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have created or modified unit tests to sufficiently cover my changes.
- [x] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

File: Test/Unit/Controller/Cart/EmailTest.php [hunk 1]
```diff
@@ -130,11 +139,12 @@ public function testExecute_noEmail()
     }
 
     /**
-     * This doesn't really seem to test invalid emails as the validateEmail method
-     * needs to be mocked. Generally just making sure that if validateEmail comes
-     * back false we error handle properly
+     * @test
      */
-    public function testExecute_invalidEmail()
+    // Note: This doesn't really seem to test invalid emails as the validateEmail method
```
Comment (matt-thomason) [inline Test/Unit/Controller/Cart/EmailTest.php]: Can you bubble this comment into the phpdoc?

File: Test/Unit/Helper/ConfigTest.php [hunk 2]
```diff
@@ -505,19 +505,19 @@ public function testGetAdditionalJS()
...
 
         $this->assertInstanceOf(ScopeConfigInterface::class, $result, 'getScopeConfig() method: not working properly');
     }
 
     /**
-     * @inheritdoc
+     * @test
      */
-    public function testGetIgnoredShippingAddressCoupons()
+    public function gtIgnoredShippingAddressCoupons()
```
Comment (matt-thomason) [inline Test/Unit/Helper/ConfigTest.php]: Small typo

File: Test/Unit/Controller/Cart/EmailTest.php (hunk 1 above)
Comment (daisy1754) [reply Test/Unit/Controller/Cart/EmailTest.php]: I actually moved it form php doc to comment - I don't think this should live in PHPDoc. PHP doc is for public API documentation. This is more about note around how this function works internally

Comment (matt-thomason) [reply Test/Unit/Controller/Cart/EmailTest.php]: Fair enough 👍

**Pull request merged**